<ui version="4.0" >
 <class>colorPicker</class>
 <widget class="QWidget" name="colorPicker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>148</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>120</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>color picker</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <property name="horizontalSpacing" >
    <number>2</number>
   </property>
   <property name="verticalSpacing" >
    <number>2</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Red:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="red" >
     <property name="maximum" >
      <number>255</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QWidget" native="1" name="bRed" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Gren:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSlider" name="green" >
     <property name="maximum" >
      <number>255</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QWidget" native="1" name="bGreen" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Blue:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSlider" name="blue" >
     <property name="maximum" >
      <number>255</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QWidget" native="1" name="bBlue" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QPushButton" name="editBtn" >
     <property name="text" >
      <string>Change</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>red</tabstop>
  <tabstop>green</tabstop>
  <tabstop>blue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
